spec.md

```diff
@@ -467,6 +485,8 @@ of this specification.
 | end-9  | `DELETE` | `/v2/<name>/manifests/<reference>`                                | `202`                              | `404`/`400`/`405`               |
 | end-10 | `DELETE` | `/v2/<name>/blobs/<digest>`                                       | `202`                              | `404`/`405`                     |
 | end-11 | `POST`   | `/v2/<name>/blobs/uploads/?mount=<digest>&from=<other_namespace>` | `201`                              | `404`                           |
+| end-12 | `HEAD`   | `/v2/<name>/blobs/<digest>`                                       | `200`                              | `404`                           |
```

Comment:
Could the value under `Method` for these endpoints be `GET` / `HEAD` rather than treating them as a separate endpoint entry.

Reply:
@dmcgowan sure, just pushed a change to that effect, thanks.